src/codegen.c

```diff
@@ -455,6 +472,9 @@ static void codegen_expr(CodegenContext *ctx, Node *expr) {
   case NODE_LITERAL:
     if (expr->literal.type == TK_NUMBER) expr->ir = ir_immediate(ctx, expr->type, expr->literal.integer);
     else if (expr->literal.type == TK_STRING) {
+      // TODO: We should probably set this name earlier, or have some
+      // way of getting this name from jujst a string index. Static
+      // variable is big bad. Valve, pls fix. Literally unplayable.
```

Comment:
How would it make strings hard to debug? :eyes:

Reply:
If a string static is named just some garbage/random nonsense by the backend, I feel like that'd be harder to debug than if the string static has a name like `__str__0` or smth

Reply:
I mean, the backend could still assign a `__str_...` name to it

Reply:
Though I suppose it’s not *that* important where that happens. I just figured it would make more sense in the backend

Reply:
I see now; I think I misinterpreted the meaning of "backend just generates a name for it" as "it's name is x8i7a879a". What you are saying is this should happen later rather than earlier, with the backend deciding what to generate when given an empty string name

Reply:
Pretty much yeah